### ISSUE TITLE
[BUG] A lot methods not found in `ffi`

### ISSUE THREAD

**Issue by marioidival** [dgrunwald/rust-cpython#153]
```~/workspace/testapp >>> cargo build                                     ±[●][master]
   Compiling cpython v0.2.0
error[E0425]: cannot find function `PyEval_ThreadsInitialized` in module `ffi`
  --> /home/kakarotto/.cargo/registry/src/github.com-1ecc6299db9ec823/cpython-0.2.0/src/pythonrun.rs:55:26
   |
55 |             assert!(ffi::PyEval_ThreadsInitialized() != 0);
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `ffi`

error[E0425]: cannot find function `PyEval_ThreadsInitialized` in module `ffi`
  --> /home/kakarotto/.cargo/registry/src/github.com-1ecc6299db9ec823/cpython-0.2.0/src/pythonrun.rs:58:26
   |
58 |             assert!(ffi::PyEval_ThreadsInitialized() == 0);
   |                          ^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `ffi`

error[E0425]: cannot find function `PyEval_InitThreads` in module `ffi`
  --> /home/kakarotto/.cargo/registry/src/github.com-1ecc6299db9ec823/cpython-0.2.0/src/pythonrun.rs:65:18
   |
65 |             ffi::PyEval_InitThreads();
   |                  ^^^^^^^^^^^^^^^^^^ did you mean `PyEval_SaveThread`?

error[E0425]: cannot find function `PyEval_InitThreads` in module `ffi`
   --> /home/kakarotto/.cargo/registry/src/github.com-1ecc6299db9ec823/cpython-0.2.0/src/lib.rs:335:10
    |
335 |     ffi::PyEval_InitThreads();
    |          ^^^^^^^^^^^^^^^^^^ did you mean `PyEval_SaveThread`?

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0425`.
error: Could not compile `cpython`.

To learn more, run the command again with --verbose.
```

**Comment by benruijl** [comment]
I assume you are using Python 3.7? https://github.com/dgrunwald/rust-cpython/issues/152

**Comment by marioidival** [comment]
@benruijl yes :thinking:

**Comment by benruijl** [comment]
You can use the trick in my linked issue, but sadly you can't publish a crate with a github dependency.

**Comment by marioidival** [comment]
@benruijl ok, I'll test with your trick! Thank you.

**Comment by dgrunwald** [comment]
I just released version 0.2.1.
